download/upgradeRecipe/upgradeRecipe7.adoc

```diff
@@ -3026,3 +3026,60 @@ public class MyPlanningSolution {
 
 }
 ----
+
+[.upgrade-recipe-minor]
+=== `Score.toInitializedScore()` deprecated for removal
+
+The `Score.toInitializedScore()` method has been deprecated in favor of `Score.withInitScore(int)`.
+
+Before in `*.java`:
+
+[source,java]
+----
+score = score.toInitializedScore();
+----
+
+After in `*.java`:
+
+[source,java]
+----
+score = score.withInitScore(0);
+----
+
+[.upgrade-recipe-minor]
+=== Various `Comparators` deprecated for removal
```

Comment:
I'd change to "Various justification `Comparators`..."

Reply:
(minor tags are collapsed by default in the output page, so the title is double important)